profiles/kentik_snmp/cisco/cisco-all-devices.yml

```diff
@@ -1133,3 +1134,17 @@ metrics:
           name: cHsrpGrpStandbyRouter
           OID: 1.3.6.1.4.1.9.9.106.1.2.1.1.14
           conversion: hextoip
+      
+  # POE stats
+  - MIB: CISCO-POWER-ETHERNET-EXT-MIB
+    metric_tags:
+      #  This object indicates used power expressed in miliwatts    
+      - column:
+          name: cpeExtMainPseUsedPower
+          OID: 1.3.6.1.4.1.9.9.402.1.3.1.4
+          conversion: milliwatts to watts
+      #  This object indicates remaining power expressed in miliwatts, this parameter is calculated as pethMainPsePower minus cpeExtMainPseUsedPower 
+      - column:
+          name: cpeExtMainPseRemainingPower
+          OID: 1.3.6.1.4.1.9.9.402.1.3.1.5
+          conversion: milliwatts to watts
```

Comment:
these should not be nested inside of the `metric_tag` attribute. The metrics are part of the [cpeExtMainPseTable](https://mibs.observium.org/mib/CISCO-POWER-ETHERNET-EXT-MIB/#cpeExtMainPseTable) and need to be configured as a mib table as seen in the [template file](https://github.com/kentik/snmp-profiles/blob/7694cd65a24e8ae35ad334209d33e041adf7731e/profiles/kentik_snmp/_template.yml#L41-L79).